### PR TITLE
add test for timestamp related values

### PR DESCRIPTION
**Pull request by wouanagaine**
connects #1

### CODE REVIEW

**Comment by wouanagaine** [comment]
@cloderic can you review please ?
to be integrated when #346 is integrated in craft-ai/learning for version 0.0.2

File: package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "craft-ai"
   ],
   "dependencies": {
-    "lodash": "4.11.1"
+    "lodash": "4.11.1",
+    "moment": "^2.13.0"
```
Comment (cloderic) [inline package.json]: Use fixed versions

File: src/DecisionTree.js [hunk 2]
```diff
@@ -48,7 +49,48 @@ let evaluateDecisionTree0_0_1 = function( node, context ) {
...
+      const day = currentDate.isoWeekday() - 1;
+      sample[key] = day;
+    }
+  });
+  sample[sampledName.timezoneKey] = timezone;
+  return sample;
+};
+
+
+export default function evaluateDecisionTree( json, context, timestamp ) {
```
Comment (cloderic) [inline src/DecisionTree.js]: maybe explicitly make it something like `evaluateDecisionTree( json, context, timestamp = undefined )`

File: src/DecisionTree.js [hunk 3]
```diff
@@ -48,7 +49,48 @@ let evaluateDecisionTree0_0_1 = function( node, context ) {
   };
 };
 
-export default function evaluateDecisionTree( json, context ) {
+
+let getSampledName = function( model ) {
+  let timeKeys = {};
+  let timezoneKey = '';
+  _.map(model, (value, key) => {
```
Comment (cloderic) [inline src/DecisionTree.js]: as you use it, it should be a `_.each`

File: src/DecisionTree.js [hunk 4]
```diff
@@ -48,7 +49,48 @@ let evaluateDecisionTree0_0_1 = function( node, context ) {
...
+};
+
+
+let generateTime = function( sampledName, currentTimestamp, timezone ) {
+  let timeKeys =  sampledName.timeKeys;
+
+  let currentDate = moment.unix(currentTimestamp).utcOffset(timezone);
+  let sample = {};
+
+  _.map(timeKeys, (type, key) => {
```
Comment (cloderic) [inline src/DecisionTree.js]: should be a `_.each`

**Comment by wouanagaine** [comment]
@cloderic I've fixed comments

**Pull request merged**